app/presenters/rejection_reasons/rejection_reasons_presenter.rb

```diff
@@ -21,15 +21,18 @@ def reasons
 
     def nested_reasons(reason)
       reason.selected_reasons.each_with_object([]) do |nested_reason, ary|
-        ary << formatted_label(nested_reason)
-        ary << nested_reason.details.text if nested_reason.details
+        if nested_reason.details
+          ary << "#{nested_reason.label}:" if render_label?(nested_reason.label, reason)
+          ary << nested_reason.details.text
+        else
+          ary << "#{nested_reason.label}."
+        end
       end
     end
 
-    def formatted_label(reason)
-      return "#{reason.label}:" if reason.details
-
-      "#{reason.label}."
+    def render_label?(label, reason)
+      nested_reasons_with_details = reason.selected_reasons.select(&:details)
+      label != 'Other' || nested_reasons_with_details.size > 1
```

Comment:
So I understand the different scenarios:
1) If we only fill in one details other box in nested reasons we don't show "other" label
2) If we fill in multiple details boxes including "other" we show the label

are there any other combinations?

Reply:
I've amended this commit as the implementation was wrong, as identified in design review. The correct condition is:

- If the 'Other' details reason/text is the only selected sub-reason then the `Other:` label should not be rendered.